### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#26904]
#### VFS team name
Decision Tools

#### VFS product name
Find a VA Form, Add Relevance, Popularity, Related to 

#### Point of Contact/Reviewers
Angela Fowler
Josh Kim

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On the Find a Form search results page, the results are located within a definition list. While this offers a way to jump to the results, it offers no way for screen readers to easily skip between the individual results. Screen reader users who are accustomed to navigating search results by heading will likely skip over them. Edit: [For more on semantic search results, reference #26363](https://github.com/department-of-veterans-affairs/va.gov-team/issues/26353)

On the Find a Form search results page, card titles under the `h2` "Frequently used VA forms" use the `b` element which [carries no semantic meaning](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b). Since they use `b` instead of a heading, the same visual hierarchy that sighted users can understand will be lost to screen reader users.

### Supporting Guidance
Category 02, Issue 04
(this section will eventually link to the VA.gov Experience Standards on Platform Website, however it's not live yet)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
The form title, I.E. "10-10EZ (esp) Instrucciones y Solicitud de Beneficios Medicos," should be contained within a level 3 heading. This has been captured in ticket 26353.

Instead of the `b` element, use a `h3` and validate the order is logical using headingsMap.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by zacharymorel** [comment]
Ref for the future for solving the issue => https://github.com/department-of-veterans-affairs/va.gov-team/issues/26068

**Comment by zacharymorel** [comment]
Ref with screenshots for the future for solving the issue => https://github.com/department-of-veterans-affairs/va.gov-team/issues/26068

**Comment by zacharymorel** [comment]
@joshkimux does this user story include adjusting the headings, exchanging Bold tags for headers, and exchanging the description list for an unordered list?

**Comment by joshkimux** [comment]
This covers adjusting headings and bold tags, but doesn't cover changing the description list to an unordered list. I believe there's a separate ticket for that 👍

**Comment by zacharymorel** [comment]
Thanks @joshkimux

**Comment by zacharymorel** [comment]
@joshkimux can you find that ticket for the search results list change? 
I don't see it here linked => https://github.com/department-of-veterans-affairs/va.gov-team/issues/26298

**Comment by joshkimux** [comment]
Here it is! I'll link it to this ticket
https://github.com/department-of-veterans-affairs/va.gov-team/issues/26353

**Comment by zacharymorel** [comment]
@joshkimux This is on staging!

**Comment by joshkimux** [comment]
<img width="1394" alt="Screen Shot 2021-08-05 at 12 56 21 PM" src="https://user-images.githubusercontent.com/14154792/128390502-4e15099c-c4aa-49c7-be8c-e95fbfa41880.png">

Closing this out. Great work @zacharymorel ! 🥳

**Comment by MarciMcGuireGCIO** [comment]
Whoo!  Thanks again @zacharymorel and @joshkimux ! ❤️

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process
